cmd/kk/apis/kubekey/v1alpha2/cluster_types.go

```diff
@@ -135,12 +135,15 @@ func (cfg *ClusterSpec) GenerateCertSANs() []string {
 		if host.Address != cfg.ControlPlaneEndpoint.Address {
 			extraCertSANs = append(extraCertSANs, host.Address)
 		}
-		if host.InternalAddress != host.Address && host.InternalAddress != cfg.ControlPlaneEndpoint.Address {
-			extraCertSANs = append(extraCertSANs, host.InternalAddress)
+		InternalIPv4Address := strings.Split(host.InternalAddress, ",")[0]
+		InternalIPv6Address := strings.Split(host.InternalAddress, ",")[1]
```

Comment:
If the ipv6 address is not configured for the hosts, Ithink it should be reported `panic`.